docker/test/Dockerfile

```diff
@@ -26,6 +26,11 @@ RUN echo '>>> update pip...' && \
     echo '>>> install requirements-dev.txt...' && \
     pip install --no-cache-dir -r requirements-dev.txt
 
+# RUN echo '>>> update pip...' && \
```

Comment:
I added this so future me doesn't have to remember how I bootstrap everything not building, but needing to partially build things so that I can fix the things so it can fully build.